.gitlab/common/test_infra_version.yml

```diff
@@ -2,6 +2,6 @@
 variables:
   # To use images from test-infra-definitions dev branches, set the SUFFIX variable to -dev
   # and check the job creating the image to make sure you have the right SHA prefix
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: "-dev"
   # Make sure to update test-infra-definitions version in go.mod as well
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 1f340fd450e3
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 4d04b8f3553c
```

Comment:
This is the build image built from DataDog/test-infra-definitions#1282.
This will need to be updated and the `-dev` suffix removed once the said PR is merged.